ops/scripts/check-sequencer.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+set -e
+
```

Comment:
Where is this script used?

Reply:
oo this needs to be deleted, will do that